webapp/app.py

```diff
@@ -1179,32 +1145,39 @@ def takeovers_index():
     methods=["GET", "POST"],
 )
 
-
 # HPE blog section
 def render_blogs():
     blogs = BlogViews(
-        api=BlogAPI(
-            session=session, thumbnail_width=555, thumbnail_height=311
-        ),
+        api=BlogAPI(session=session, thumbnail_width=555, thumbnail_height=311),
         tag_ids=[4307],
         per_page=3,
         blog_title="HPE blogs",
     )
     hpe_articles = blogs.get_tag("hpe")
-    return flask.render_template(
-        "/hpe/index.html", blogs=hpe_articles["articles"]
-    )
-
+    return flask.render_template("/hpe/index.html", blogs=hpe_articles["articles"])
 
 app.add_url_rule("/hpe", view_func=render_blogs)
 
+# Public-cloud blog section
+# tag_ids:
+# public-cloud - 1350, aws - 1205, azure - 1748, google-cloud - 4191, ubuntu-on-aws - 4478, ubuntu-on-gcp - 4387, ubuntu-on-azure - 4540
+def render_public_cloud_blogs():
```

Comment:
We actually already have a module responsible for rendering blogs https://github.com/canonical/latest-news. It isn't capable of taking multiple ID's, but we could definitely update it to do so. I think this is preferable than writing a new python func each time we need to call a different blog post. This may also alleviate the presence of japanese blogs